INSTALL.md

```diff
@@ -53,6 +53,8 @@ source ../tools/activate.fish  # for fish
 
 This command builds (or re-builds) `vargo`, our cargo wrapper, and adds it to the `PATH` for the current shell.
 
+If you use [direnv](https://direnv.net/), then this activation is performed automatically (i.e., you don't need to `source ../tools/activate`; instead, `vargo` will automatically be in your `PATH` as long as you are in the `source` subdirectory or any directory under it).
```

Comment:
I'd prefer we kept these instructions as streamlined as possible for all users, and this paragraph is only relevant to direnv users.

What about we added a short note earlier (line 45):

```
First, activate the development environment with one of the following: (This is automatic for [direnv](https://direnv.net/) users[^1])
```

that points to a footnote:

```
[^1]: If you are a [direnv](https://direnv.net/) user this activation is performed automatically, i.e. you don't need to `source ../tools/activate`; instead, `vargo` will automatically be in your `PATH` as long as you are in the `source` subdirectory.
```

(GitHub will render it as an actual footnote.)

Reply:
Good point, updated